eng/pipelines/common/global-build-job.yml

```diff
@@ -31,6 +31,9 @@ jobs:
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
     workspace:
       clean: all
+    ${{ if ne(parameters.enableRichCodeNavigation, '') }}:
```

Comment:
let's default this parameter to false in this template by adding it to the parameters list and then do a boolean comparison:

```yml
${{ if eq(parameters.enableRichCodeNavigation, true) }}:
```

Reply:
Great idea! Done.